content/pt/docs/contributing/pull-requests.md

```diff
@@ -0,0 +1,573 @@
+  Clique no botão
+  [_Fork_](https://github.com/open-telemetry/opentelemetry.io/fork) na parte
+  superior do repositório e clone o _fork_ localmente. Quando estiver pronto,
+  envie um PR para o repositório upstream.
+- **Não trabalhe a partir da branch `main`** do seu _fork_, mas crie uma
+  _branch_ específica para o PR.
+- Certifique-se de que os mantenedores estão
+  [autorizados a aplicar alterações ao seu _pull request_](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
+
+### Alterações dos revisores
+
+Às vezes, os revisores fazem _commits_ no seu _pull request_. Antes de fazer
+qualquer outra alteração, busque esses _commits_.
+
+1. Busque _commits_ do seu _fork_ remoto e faça o _rebase_ em sua _branch_ de
+   trabalho:
+
+   ```shell
+   git fetch origin
+   git rebase origin/<seu-nome-da-branch>
+   ```
+
+1. Após o _rebase_, execute um _force-push_ contendo as novas alterações para
+   seu _fork_:
+
+   ```shell
+   git push --force-with-lease origin <seu-nome-da-branch>
+   ```
+
+Você também pode resolver conflitos de _merge_ pela interface do GitHub.
+
+### Conflitos de merge e rebase
+
+Se outro colaborador fizer alterações no mesmo arquivo em outro PR, isso pode
+criar um conflito de _merge_. Você deve resolver todos os conflitos de _merge_
+em seu PR.
+
+1. Atualize seu _fork_ e faça _rebase_ em sua branch local:
+
+   ```shell
+   git fetch origin
+   git rebase origin/<seu-nome-da-branch>
+   ```
+
+   Em seguida, execute um _force-push_ contendo as alterações para seu _fork_:
+
+   ```shell
+   git push --force-with-lease origin <seu-nome-da-branch>
+   ```
+
+1. Busque alterações do `upstream/main` do repositório
+   `open-telemetry/opentelemetry.io` e faça _rebase_ de sua _branch_:
+
+   ```shell
+   git fetch upstream
+   git rebase upstream/main
+   ```
+
+1. Inspecione os resultados do _rebase_:
+
+   ```shell
+   git status
+   ```
+
+   Isso resulta em vários arquivos marcados como conflitantes.
+
+1. Abra cada arquivo em conflito e procure os marcadores de conflito: `>>>`,
+   `<<<`, e `===`. Resolva o conflito e exclua o marcador de conflito.
+
+   Para mais informações, veja
+   [Como os conflitos são apresentados](https://git-scm.com/docs/git-merge#_how_conflicts_are_presented).
+
+1. Adicione os arquivos ao conjunto de alterações:
+
+   ```shell
+   git add <nome_do_arquivo>
+   ```
+
+1. Continue o _rebase_:
+
+   ```shell
+   git rebase --continue
+   ```
+
+1. Repita as etapas 2 a 5 conforme necessário.
+
+   Após aplicar todos os _commits_, o comando `git status` mostra que o _rebase_
+   está completo.
+
+1. Execute um _force-push_ da _branch_ para seu _fork_:
+
+   ```shell
+   git push --force-with-lease origin <seu-nome-da-branch>
+   ```
+
+   O _pull request_ não mostra mais conflitos.
+
+### Requisitos de mesclagem (_merge_)
```

Comment:
Heading ID missing here and a few other places. Please add them for all headings.